<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>617</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btn_home">
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_img">
             <property name="text">
              <string>图像</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_video">
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_other">
             <property name="text">
              <string>预留</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_set">
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="page_home">
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>60</y>
               <width>69</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_video">
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>230</y>
               <width>69</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_other">
            <widget class="QWidget" name="widget_side" native="true">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>90</y>
               <width>171</width>
               <height>161</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QPushButton" name="btn_inside">
                <property name="text">
                 <string>in side</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btn_click">
                <property name="text">
                 <string>click</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>160</y>
               <width>81</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>侧边</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_set">
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>250</x>
               <y>330</y>
               <width>69</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>set</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_img">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QTabWidget" name="tabWidget_img">
               <property name="tabPosition">
                <enum>QTabWidget::West</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <property name="iconSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <widget class="QWidget" name="tab_test">
                <attribute name="title">
                 <string>测试</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="label_img_src_test">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid black;</string>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_img_dst_test">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid red;</string>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QPushButton" name="btn_img_open_file">
                       <property name="text">
                        <string>open</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_img">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="spinBox_img"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_debug">
                <attribute name="title">
                 <string>调试</string>
                </attribute>
                <widget class="QPushButton" name="btn_img_open">
                 <property name="geometry">
                  <rect>
                   <x>500</x>
                   <y>30</y>
                   <width>71</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>open</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_img_binaryzation">
                 <property name="geometry">
                  <rect>
                   <x>590</x>
                   <y>70</y>
                   <width>71</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>二值化</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_img_gray">
                 <property name="geometry">
                  <rect>
                   <x>500</x>
                   <y>70</y>
                   <width>71</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>灰度图</string>
                 </property>
                </widget>
                <widget class="QScrollArea" name="scrollArea_img_src">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>20</y>
                   <width>361</width>
                   <height>211</height>
                  </rect>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>359</width>
                    <height>209</height>
                   </rect>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_img_src">
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="text">
                      <string>原式图像</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
                <widget class="QScrollArea" name="scrollArea_img_dst">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>260</y>
                   <width>361</width>
                   <height>231</height>
                  </rect>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_2">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>359</width>
                    <height>229</height>
                   </rect>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_6">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_img_dst">
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="text">
                      <string>结果图像</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
                <widget class="QComboBox" name="comboBox_img_feature">
                 <property name="geometry">
                  <rect>
                   <x>580</x>
                   <y>140</y>
                   <width>79</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <item>
                  <property name="text">
                   <string>功能</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>水平翻转</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>垂直翻转</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>灰度显示</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>去紫边</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>除雾模式</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>增强纹理</string>
                  </property>
                 </item>
                </widget>
                <widget class="QPushButton" name="btn_img_open_dir">
                 <property name="geometry">
                  <rect>
                   <x>580</x>
                   <y>190</y>
                   <width>60</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>open dir</string>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
